assets/js/analyzer.js

```diff
@@ -241,6 +243,8 @@ function handleAnalyzeErrorResponse(xOptions, error) {
     $('#morphAnalyzerOutput').text(error).removeClass('blurred');
 }
 
+/*:: export {analyzerData, analyzers, getAnalyzers, populateAnalyzerList, populateSecondaryAnalyzerList} */
```

Comment:
Fine to have a empty line between these but I don't think all the files do.

Reply:
I'll make it so.